.travis.yml

```diff
@@ -1,6 +1,9 @@
 branches:
   only:
     - develop
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y imagemagick
```

Comment:
This won't work :(. We're using Travis containerised builds so sudo isn't available. It would be nice to keep containerised builds because they're shiny and underutilised, but if we can't work around it we just need to remove `sudo: false` from `.travis.yml`